components/dashboard/src/admin/TeamDetail.tsx

```diff
@@ -151,6 +225,89 @@ export default function TeamDetail(props: { team: Team }) {
                     ))
                 )}
             </ItemsList>
+            <Modal
+                visible={editSpendingLimit}
+                onClose={() => setEditSpendingLimit(false)}
+                title="Update Usage Limit"
```

Comment:
**issue**: When trying to update the usage limit for a team I get the following console error and the limit does not update. Is this known or expected?

```js
Uncaught (in promise) Error: Failed to get Stripe Subscription ID for 'team:0faf0b45-1e5f-4c3c-b5ef-6bbe22f1da9f'
```